### PR TITLE
MUIC-618: [Android] - Extract colors for header title and home button

### PR DESCRIPTION
**Pull request by deniss-mc**
# https://glia.atlassian.net/browse/MUIC-618

### CODE REVIEW

**Review by dukhovnyi — APPROVED** [review]
for me, it looks OK, but please wait for @AndriiHorishniiMOC  review

File: widgetssdk/src/main/java/com/glia/widgets/view/header/AppBarView.java
Comment (jakob-mc) [inline widgetssdk/src/main/java/com/glia/widgets/view/header/AppBarView.java:65]: this method is getting very long. can it be broken down? like `setTitleTextColor()`

File: widgetssdk/src/main/java/com/glia/widgets/view/header/AppBarView.java
Comment (jakob-mc) [inline widgetssdk/src/main/java/com/glia/widgets/view/header/AppBarView.java:117]: unnecessary empty line

**Review by jakob-mc — CHANGES_REQUESTED** [review]

**Comment by dukhovnyi** [comment]
Andrii is busy, ok to merge without his review when it will be ready.

File: widgetssdk/src/main/java/com/glia/widgets/view/header/AppBarView.java
Comment (deniss-mc) [reply widgetssdk/src/main/java/com/glia/widgets/view/header/AppBarView.java:117]: Please validate in MUIC-619

**Pull request closed without merging**
